ftw/noticeboard/browser/templates/notice.pt

```diff
@@ -20,6 +20,12 @@
     </head>
 
     <div metal:fill-slot="main">
+        <div class="noticeboard-actions">
+            Aktionen: <a class="edit-notice" tal:condition="view/can_edit" tal:attributes="href python:context.absolute_url() + '/edit'" i18n:translate="edit">Edit</a> | 
+            <a class="edit-notice" tal:condition="view/can_edit" tal:attributes="href python:context.absolute_url() + '/folder_contents'" i18n:translate="edit_images">Edit images</a> | 
+            <a class="edit-notice" tal:condition="view/can_edit" tal:attributes="href python:context.absolute_url() + '/delete_confirmation'" i18n:translate="delete_images">Delete notice</a>
```

Comment:
Is it intentional that they all have the class `edit-notice` even though one is a `delete` etc.